cmd/healthz/main.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"context"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/gravitational/satellite/healthz/clusterstatus"
+	"github.com/gravitational/satellite/healthz/config"
+	"github.com/gravitational/satellite/healthz/service"
+	"github.com/gravitational/satellite/healthz/utils"
+	"github.com/gravitational/trace"
+)
+
+func run() error {
+	status := clusterstatus.NewClusterStatus()
+	cfg := &config.Config{}
+
+	if err := config.ParseCLIFlags(cfg); err != nil {
+		return trace.Wrap(err)
+	}
+
+	if err := utils.SetupLogging(cfg.LogLevel); err != nil {
+		return trace.Wrap(err)
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+
+	go func() {
+		exitSignals := make(chan os.Signal, 1)
+		signal.Notify(exitSignals, syscall.SIGTERM, syscall.SIGINT)
+
+		select {
+		case sig := <-exitSignals:
+			log.Infof("signal: %v", sig)
+			cancel()
```

Comment:
Ignored all signals here https://github.com/gravitational/satellite/pull/5/files#diff-ab19c86a3dceb0020b670ef4f149bc24R48, so only TERM and INT are cought